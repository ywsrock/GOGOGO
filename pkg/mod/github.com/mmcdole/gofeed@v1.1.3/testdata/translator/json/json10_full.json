{
  "version": "1.0",
  "title": "title",
  "author": {
    "avatar": "https://sample-feed-author.com/me.png",
    "name": "author_name",
    "url": "https://sample-feed-author.com"
  },
  "description": "description",
  "expired": false,
  "favicon": "https://sample-json-feed.com/favicon.png",
  "feed_url": "https://sample-json-feed.com/feed.json",
  "home_page_url": "https://sample-json-feed.com",
  "icon": "https://sample-json-feed.com/icon.png",
  "next_url": "https://sample-json-feed.com/feed.json?next=500",
  "user_comment": "user_comment",
  "items": [
    {
      "attachments": [
        {
          "duration_in_seconds": 100,
          "mime_type": "audio/mpeg",
          "size_in_bytes": 100,
          "title": "title",
          "url": "https://sample-json-feed.com/attachment"
        }
      ],
      "author": {
        "avatar": "https://sample-feed-author.com/me.png",
        "name": "author_name",
        "url": "https://sample-feed-author.com"
      },
      "banner_image": "https://sample-json-feed.com/banner_image.png",
      "content_html": "<p>content_html</p>",
      "content_text": "content_text",
      "date_modified": "2019-10-12T07:20:50.52Z",
      "date_published": "2019-10-12T07:20:50.52Z",
      "external_url": "https://sample-json-feed.com/external",
      "id": "id",
      "image": "https://sample-json-feed.com/image.png",
      "summary": "summary",
      "tags": ["tag1", "tag2"],
      "title": "title",
      "url": "https://sample-json-feed.com/id"
    }
  ]
}
