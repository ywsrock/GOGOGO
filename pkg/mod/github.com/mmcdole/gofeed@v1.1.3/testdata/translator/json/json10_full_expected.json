{
  "feedVersion": "1.0",
  "feedType": "json",
  "feedLink": "https://sample-json-feed.com/feed.json",
  "title": "title",
  "author": {
    "avatar": "https://sample-feed-author.com/me.png",
    "name": "author_name",
    "url": "https://sample-feed-author.com"
  },
  "authors": [
    {
      "avatar": "https://sample-feed-author.com/me.png",
      "name": "author_name",
      "url": "https://sample-feed-author.com"
    }
  ],
  "description": "description",
  "link": "https://sample-json-feed.com",
  "image": {
    "url": "https://sample-json-feed.com/icon.png"
  },
  "updated": "2019-10-12T07:20:50.52Z",
  "updatedParsed": "2019-10-12T07:20:50.52Z",
  "published": "2019-10-12T07:20:50.52Z",
  "publishedParsed": "2019-10-12T07:20:50.52Z",
  "links": [
    "https://sample-json-feed.com",
    "https://sample-json-feed.com/feed.json"
  ],
  "items": [
    {
      "guid": "id",
      "title": "title",
      "link": "https://sample-json-feed.com/id",
      "links": [
        "https://sample-json-feed.com/id",
        "https://sample-json-feed.com/external"
      ],
      "content": "<p>content_html</p>",
      "updated": "2019-10-12T07:20:50.52Z",
      "updatedParsed": "2019-10-12T07:20:50.52Z",
      "published": "2019-10-12T07:20:50.52Z",
      "publishedParsed": "2019-10-12T07:20:50.52Z",
      "description": "summary",
      "categories": [
        "tag1",
        "tag2"
      ],
      "enclosures": [
        {
          "length": "100",
          "type": "audio/mpeg",
          "url": "https://sample-json-feed.com/attachment"
        }
      ],
      "author": {
        "avatar": "https://sample-feed-author.com/me.png",
        "name": "author_name",
        "url": "https://sample-feed-author.com"
      },
      "authors": [
        {
          "avatar": "https://sample-feed-author.com/me.png",
          "name": "author_name",
          "url": "https://sample-feed-author.com"
        }
      ],
      "image": {
        "url": "https://sample-json-feed.com/image.png"
      }
    }
  ]
}